### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by fsteamID**
fix

### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [x] This is Software as a Service not self hosted
 - [x] It has a free tier not just a free trial
 - [x] The submission mentions what is free
 - [x] The submission is not already present in the list
 - [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1083,6 +1083,7 @@ Table of Contents
   * [is-a.dev](https://www.is-a.dev/) — Free is-a.dev subdomain for developers.
   * [is-a-good.dev](https://is-a-good.dev) — A free is-a-good-dev subdomain for developers.
   * [cluster.ws & wip.la](https://github.com/Olivr/free-domain) — Free cluster.ws and wip.la subdomains for developers.
+  * [Dee.al](https://domain.dee.al) — Free .dee.al,.gal.aw,.devz.id instant domain for your blog site & Other, ENJOY
```
Comment (ripienaar) [inline README.md:1086]: The github readme does not seem to be useful teaching people how to use this, let us know once this is improved to its usable

**Comment by FatGrizzly** [comment]
Please have English docs too.


https://github.com/dee-al/domain.dee.al/tree/main/domain/dee.al

**Comment by fsteamID** [comment]
I have updated the document to an international language

**Comment by ripienaar** [comment]
Thank you

**Pull request merged**